plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/RepositoryCMAComponent.groovy

```diff
@@ -103,6 +104,7 @@ extends DirectComponentSupport
         name: input.name,
         type: input.type,
         format: input.format,
+        online: input.facet(ViewFacet).online,
```

Comment:
ViewFacet is always available. facet() call will throw an exception if facet is not available